src/cpl/nuopc/lnd_import_export.F90

```diff
@@ -1294,20 +1294,27 @@ subroutine ReadCapNamelist( NLFilename )
     ! ----------------------------------------------------
     ! Read in tne namelist for CTSM nuopc cap level items
     ! ----------------------------------------------------
+    use ESMF             , only : ESMF_VMGetCurrent, ESMF_VMBroadcast, ESMF_VM
     use clm_nlUtilsMod   , only : find_nlgroup_name
-    use shr_mpi_mod      , only : shr_mpi_bcast
-    use spmdMod          , only : mpicom
     use abortutils       , only : endrun
     use shr_log_mod      , only : errMsg => shr_log_errMsg
     ! !ARGUMENTS:
     character(len=*), intent(IN) :: NLFilename   ! Namelist filename
     ! !LOCAL VARIABLES:
     integer            :: nu_nml                           ! unit for namelist file
     integer            :: nml_error                        ! namelist i/o error flag
+    integer            :: rc                               ! ESMF return code
```

Comment:
- [ ] I'm thinking that rc should be a subroutine output argument that is checked by the caller so that errors can propagate up the call chain; do you agree?

Reply:
done